### ISSUE TITLE
Format converting during spell-check

### ISSUE THREAD

**Issue by Yamato-ua** [SubtitleEdit/subtitleedit#2185]
Hello.

It's odd but when I do final spell-checking (the Word one), the subtitle format changes from Structured titles to SubRip. When trying to rollback to the version after the spell-check but before the format change, it's still SubRip. I'm guessing these two processes are somehow tied into one.

It's beta 3.5.1 build 9.

upd. To be precise, it happens the moment you click OK at the bottom of the main spell-checking window.

**Comment by niksedk** [comment]
Hm, that does not happen here... is it only with one sub or with all subs.

**Comment by Yamato-ua** [comment]
I've tried a couple episodes so far. 
Maybe it has something to do with the language. I use Russian spell and grammar check (Microsoft Word).

Will try to find more files to test.

**Comment by Yamato-ua** [comment]
I've been testing some more. 

Happens with every Structured titles file I've been able to find. Tried Russian and English US dictionaries with English and Russian text in UTF-8 and 1251:Cyrillic encoding.
Happens the moment you click OK in the end to apply changes, additions to user dic etc.

Changing it back is simple enough, but the thing is it then resets the position marker to SE default C1Y00.
